python/ray/experimental/state/common.py

```diff
@@ -516,6 +547,35 @@ class ClusterEventState(StateSchema):
     custom_fields: Optional[dict] = state_column(filterable=False, detail=True)
 
 
+class Humanify:
+    """A class containing default methods to convert units into a human readable string."""
+
+    def timestamp(x: float):
+        """Converts miliseconds to a datetime object."""
+        return str(datetime.datetime.fromtimestamp(x / 1000))
+
+    def memory(x: int):
+        """Converts raw bytes to a human readable memory size."""
+        if x >= 2**30:
+            return str(x / (2**30)) + " GiB"
+        elif x >= 2**20:
+            return str(x / (2**20)) + " MiB"
+        elif x >= 2**10:
+            return str(x / (2**10)) + " KiB"
+        return str(x) + " B"
+
+    def duration(x: int):
+        """Converts miliseconds to a human readable duration."""
+        return str(datetime.timedelta(milliseconds=x))
+
+    def events(events: List[dict]):
+        """Converts a list of task events into a human readable format."""
+        for event in events:
```

Comment:
Regarding your comment about event customizability for humanify:

the input events here right now is only used for TaskState.events, it still has the created_ms field.

My thinking was Humanify would be modified according to use case (with feedback from users), and future contributors will have a better sense of how they want to interpret the CLI output. I just made a wild guess that TaskState.events.created_ms might benefit from being humanified, and left this implementation as a future reference in case maybe another field gets added to TaskState.events and contributors can adjust the code to format it however they need.

Reply:
Yes, what you mentions makes sense. 
So all timestamp for events would have "_ms" suffix right? Could we just timestamp humanify those by default?

Reply:
Hmm, actually, I think it's fine for now. Although I feel having the constrain of `created_ms` is a bit too specific and constraining. But this is fine.